Makefile

```diff
@@ -417,7 +417,7 @@ dm_unit_test_in_verify_ci: check_failpoint_ctl tools/bin/gotestsum tools/bin/goc
 	|| { $(FAILPOINT_DISABLE); exit 1; }
 	tools/bin/gocov convert "$(DM_TEST_DIR)/cov.unit_test.out" | tools/bin/gocov-xml > dm-coverage.xml
 	$(FAILPOINT_DISABLE)
-	@bash <(curl -s https://codecov.io/bash) -F dm -f $(DM_TEST_DIR)/cov.unit_test.out -t $(TICDC_CODECOV_TOKEN)
+	@bash <(curl -s https://codecov.io/bash) -F unit,dm -f $(DM_TEST_DIR)/cov.unit_test.out -t $(TICDC_CODECOV_TOKEN)
```

Comment:
what's the meaning of "unit" 🤔

Reply:
Use flags management for test type dim.
- unit: means coverage from unit testing.
- integration: means coverage from integration testing.
- e2e: means coverage from end to end testing.
Ref: https://github.com/PingCAP-QE/ci/issues/2171

Reply:
Just make the word short and simple. 😆